docusaurus/docs/internal/code-base-works/localization.md

```diff
@@ -0,0 +1,127 @@
+# Localization
+
+## Internationalization (i18n)
+
+Any code producing messages, labels, numbers, dates, times, and the like should use the `i18n` store and translation strings to generate and format them instead of hardcoding English or American-isms anywhere.   Messages and number formatting uses [ICU templating](https://formatjs.io/docs/intl-messageformat) for very powerful pluralization and customizing.
+
+The `assets/translations` dir stores a YAML file with translations for each supported language.
```

Comment:
This is a repetition of what is written above.

Reply:
Different areas. One is internal docs, the other is extension docs. Resolving the conversation